scheduler/src/cook/tools.clj

```diff
@@ -249,6 +249,9 @@
                     (let [resource (keyword (name (:resource/type r)))]
                       (condp contains? resource
                         #{:cpus :mem :gpus} (assoc m resource (:resource/amount r))
+                        #{:disk} (assoc m :disk (cond-> {:request (:resource.disk/request r)}
```

Comment:
Worth a comment here: 
"" We add these additional disk resources here so they're available for fenzo binpacking. so we don't overfill a node with disk.""